File: eAirFlow.Model/Models/Seat.cs
using System;
using System.Collections.Generic;

namespace eAirFlow.Model.Models
{

    public class Seat
    {
        public int SeatId { get; set; }

        public int? AirplaneId { get; set; }

        public string? SeatNumber { get; set; }

        public int? SeatClassId { get; set; }

    }
}

File: UI/eairflow_mobile/lib/models/seat.dart
class Seat {
  int? seatId;
  int? airplaneId;
  String? seatNumber;
  int? seatClassId;

  Seat({this.seatId, this.airplaneId, this.seatNumber, this.seatClassId});

  factory Seat.fromJson(Map<String, dynamic> json) => Seat(
    seatId: json["seatId"],
    airplaneId: json["airplaneId"],
    seatNumber: json["seatNumber"],
    seatClassId: json["seatClassId"],
  );
}
